### PR TITLE
WIP: Governance chapter

### PR DESCRIPTION
**Pull request by HeidiSeibold**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -52,6 +52,7 @@ nav:
       - Community:
           - "How to contribute": Contributing.md
           - "Communication Channels": Communication-Channels.md
+          - "Governance": Governance.md
```
Comment (joaquinvanschoren) [inline mkdocs.yml:55]: The Community submenu is no longer there now we merged everything in the Contributing page. I would move this page one level up, e.g. before or after Terms.

File: docs/Governance.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+The Steering Committee consists of community representatives and partner representatives.
+
+Community representatives of the SC are nominated by a core contributor. A nomination will result in a discussion that cannot take more than a month and then a vote by the core contributors which will stay open for a week. Partner institutions who enter a collaboration agreement with the OpenML Foundation can nominate a representative on the Steering Committee, if so agreed in the collaboration agreement.
+
+SC membership votes are subject to a two-third majority of all cast votes as well as a simple majority approval of all the current SC members. When decisions are escalated to the steering committee (see the decision making process below), and no consensus can be found within a month, the SC can decide with a simple majority vote. In addition, to ensure the community character of the OpenML project, community representatives always carry at least 51% of the vote.
```
Comment (joaquinvanschoren) [inline docs/Governance.md:36]: These aspects have changed a bit. Maybe come back to this when the final version is approved?

**Review by joaquinvanschoren — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (HeidiSeibold) [reply mkdocs.yml:55]: :+1:

File: docs/Governance.md (hunk 2 above)
Comment (HeidiSeibold) [reply docs/Governance.md:36]: Yes, agreed

Comment (HeidiSeibold) [reply docs/Governance.md:36]: I set the PR to WIP to show that we wait for this. Is that ok?

Comment (joaquinvanschoren) [reply docs/Governance.md:36]: ok

**Comment by joaquinvanschoren** [comment]
I updated the description with the latest version and merged them. Thanks!

**Pull request closed without merging**